api/app/services/notifications/unsubscribe.rb

```diff
@@ -0,0 +1,16 @@
+module Notifications
+  class Unsubscribe < ActiveInteraction::Base
+    string :token
+
+    validates :token, presence: true
+
+    def execute
+      user_id = UnsubscribeToken.verify(token).dig(:user_id)
+      user = User.find user_id
```

Comment:
What if we can't find a user here for some reason? Should we handle the record not found exception as well? Or add an error if there's no user?